canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -74,6 +76,12 @@ class ToolTip extends React.Component {
 				}
 				if (tooltipTrigger && tooltip) {
 					this.updateTooltipLayout(tooltip, tooltipTrigger, tooltip.getAttribute("direction"));
+					if (tooltip.offsetHeight > 20 && this.state.singleLineTooltip) {
+						this.setState({ singleLineTooltip: false });
+					} else if (tooltip.offsetHeight <= 48 && !this.state.singleLineTooltip) {
+						// Multi-line tooltip initially but after resizing right flyout panel, tooltip is in single line
+						this.setState({ singleLineTooltip: true });
+					}
```

Comment:
It would be good to have a comment explaining why you are doing this, how it works, like we discussed?

Reply:
Added explanation of toggle between multi-line and single line tooltip in `else if` condition.